product-service/sql/DML.sql

```diff
@@ -0,0 +1,27 @@
+INSERT INTO products (title, description, author, price)
+VALUES ('Do Androids Dream of Electric Sheep?',
+        'It was January 2021, and Rick Deckard had a license to kill. Somewhere among the hordes of humans out there, lurked several rogue androids. Deckard''s assignment--find them and then...\"retire\" them. Trouble was, the androids all looked exactly like humans, and they didn''t want to be found!',
+        'Philip K. Dick', 24),
+       ('The Curious Incident of the Dog in the Night-Time',
+        'Christopher John Francis Boone knows all the countries of the world and their capitals and every prime number up to 7,057. He relates well to animals but has no understanding of human emotions. He cannot stand to be touched. And he detests the color yellow.',
+        'Mark Haddon', 10),
+       ('I Was Told There''d Be Cake: Essays', 'From the author of the novel, The Clasp, hailed by Michael Chabon,
+        Heidi Julavits, and J.Courtney Sullivan.Wry, hilarious, and profoundly genuine, this debut collection of literary essays from Sloane Crosley is a celebration of fallibility and haplessness in all their glory',
+        'Sloane Crosley', 23),
+       ('To Kill a Mockingbird',
+        'The unforgettable novel of a childhood in a sleepy Southern town and the crisis of conscience that rocked it, To Kill A Mockingbird became both an instant bestseller and a critical success when it was first published in 1960. It went on to win the Pulitzer Prize in 1961 and was later made into an Academy Award-winning film, also a classic.',
+        'Harper Lee', 15),
+       ('The Unbearable Lightness of Being',
+        'In The Unbearable Lightness of Being, Milan Kundera tells the story of a young woman in love with a man torn between his love for her and his incorrigible womanizing and one of his mistresses and her humbly faithful lover. This magnificent novel juxtaposes geographically distant places, brilliant and playful reflections, and a variety of styles, to take its place as perhaps the major achievement of one of the world’s truly great writers.',
+        'Milan Kundera', 23),
+       ('The Elephant Tree',
+        'Mark Fallon is an overworked detective investigating a spate of attacks at a string of high profile city centre nightclubs. Scott is a dejected 24 year old struggling to make ends meet working for his brother and supplementing his income with a small-scale drug dealing operation. Angela is an attractive 23 year old, raised by her father, a career criminal and small time drug dealer who supplies Scott with cannabis. This is a chilling tale spanning a few months in the lives of Scott and Angela, where realizations about the present combine with shocking revelations from the past leading to an apocalyptic climax where they no longer know whom they can trust',
+        'R.D. Ronald', 18);
```

Comment:
You can use returning id in the end of query

Reply:
I use it in create method (https://github.com/trigolos/nodejs-aws-be/pull/2/files#diff-c995bf41eaeca8ddc4ccd5d9775031e3c938eb42b9af0281f0634b47f8d4cd14R7), but how to use it inside sql idk yet